vite.config.mts

```diff
@@ -117,7 +117,9 @@ export default defineConfig(({ command, mode }) => {
 			rollupOptions: {
 				input: {
 					...glob.sync(path.resolve(BASE_PATH, '**/index.html').replace(/\\/g, '/')).reduce<Record<string, string>>((acc, cur) => {
-						const name = path.relative(__dirname, cur);
+						const name = process.platform === 'win32'
```

Comment:
I think this isn't needed, and can just use `path.normalize`?
https://nodejs.org/api/path.html#pathnormalizepath

Reply:
Or perhaps do: `path.relative(__dirname, cur).split(path.sep).join('/')`
That should always split the path based on the OS separator, and end up with a UNIX path

Reply:
normalize does not work out of box, the latter works better. Please verify mac/linux after this change